### DISCUSSION TITLE
1.29.1

### DISCUSSION THREAD

**Discussion by Koenkk** [Koenkk/zigbee2mqtt#15976] [General]
# Fixes
- #15747 Fix Home Assistant `Template variable warning: 'dict object' has no attribute 'latest_version' when rendering '{{ value_json['update']['latest_version'] }}'` error (@Koenkk)
- #15873 Fix Xiaomi covers broken in Home Assistant (@Koenkk)
- [#5246](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5246) Fix OTA update check broken (@rccoleman)
- [#5255](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5255) Fix Sinopé TH1400ZB configure failing (@MattL0)
- [#5249](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5249) Fix configure of some Hue lights failing (@zegl)
- #13600 Fix SONOFF SNZB-0X devices disconnecting from network (@blacknell)
- #15902 Fix state of TuYa TS110E_1gang_1 not updating (@Koenkk)
- #15962 Fix NodOn SIN-4-1-20 not controllable (@Koenkk)
- #15875 Fix `INSUFFICIENT_SPACE` error for Ubisys devices (@sjorge)
- #15971 Fix Iris 3460-L configure failing (@shackrat)

<hr /><em>This discussion was created from the release <a href='https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.29.1'>1.29.1</a>.</em>

**Comment by cloudbr34k84** [comment]
Upgrading now

**Comment by gras55** [comment]
Issue https://github.com/Koenkk/zigbee2mqtt/issues/15902 still not fixed sadly

**Reply by cloudbr34k84** [reply]
Are you having issues with devices dropping off, devices not syncing, or synced but when I try and say toggle a light switch they don't do anything??

**Comment by didier-guerin85** [comment]
I migrated from 1.24.0 to 1.29.1 and I no longer display the workdays schedule and holidays schedule.

Sinon OK
![Capture d’écran 2022-12-31 165132](https://user-images.githubusercontent.com/100363194/211342563-b86cb8ab-7a47-4846-8bb4-695dca9fc6a9.png)
![Capture d’écran 2022-12-31 165344](https://user-images.githubusercontent.com/100363194/211342715-6734e8d6-6954-4c54-860a-1b29dec84096.png)
![Capture d’écran 2023-01-07 153054](https://user-images.githubusercontent.com/100363194/211342833-5c6a8604-dd48-4f29-9b23-7dc0d42ed6b0.png)

**Comment by ondras76** [comment]
#13600 Fix SONOFF SNZB-0X devices disconnecting from network - not fixed still, the problem is, the value is not updating on temp. sensor.

**Reply by blacknell** [reply]
My SNZB-04's are still falling off the network. Even though the battery reporting intervals are now 3600/7200 I'm not seeing changes to the last seen. LQI and battery are high. I think the -04 variant is misbehaving

**Comment by blacknell** [comment]
I am noticing that Z2M is (over time) updating the reporting intervals on my SNZB-0X devices. Since they are battery devices this needs to happen when the device is next connected (such as sending an open/close or temperature). The coordinator can't initiate an update at will.

I was expecting the new reporting intervals to be updated only when repairing (or at least forcing a soft re-pairing/re-admission) so I wasn't expecting it - @Koenkk is this expected behaviour?

**Reply by sjorge** [reply]
Sadly, yes. When the configure() changes the next time the device sends a message a reconfigure is triggered. This is to fix/improve devices over time. 

I have a script i run every update to (re)apply some custom reporting settings. But having a ‘lock’ checkbox or something for reporting settings the user manually changed would be nice.

I think i filed a feature request for it at one point, but there was no interest in it aside from me.

**Reply by Koenkk** [reply]
@blacknell as @sjorge explained, when the `configure` changes yes (which I did with your fix for this device)

**Comment by blacknell** [comment]
So I waited for my SNZB-04 to fall off the network (despite the fact the reporting had been manually set) and then attempt to update when I open/closed the window.

I got this error message in the log

```
Debug 2023-01-12 08:11:21 Received Zigbee message from 'contact_bathroom', type 'commandStatusChangeNotification', cluster 'ssIasZone', data '{"extendedstatus":0,"zonestatus":1}' from endpoint 1 with groupID 0
Info 2023-01-12 08:11:21 Configuring 'contact_bathroom'
Debug 2023-01-12 08:11:21 Device 'contact_bathroom' reconnected
Info 2023-01-12 08:11:21 MQTT publish: topic 'zigbee2mqtt/contact_bathroom/availability', payload 'online'
Info 2023-01-12 08:11:21 MQTT publish: topic 'zigbee2mqtt/contact_bathroom', payload '{"battery":100,"battery_low":false,"contact":false,"last_seen":"2023-01-12T08:11:21+00:00","linkquality":61,"tamper":false,"voltage":3000}'
Debug 2023-01-12 08:11:23 Retrieving state of 'contact_bathroom' after reconnect
Debug 2023-01-12 08:11:25 Received Zigbee message from 'contact_bathroom', type 'commandStatusChangeNotification', cluster 'ssIasZone', data '{"extendedstatus":0,"zonestatus":0}' from endpoint 1 with groupID 0
Info 2023-01-12 08:11:25 MQTT publish: topic 'zigbee2mqtt/contact_bathroom', payload '{"battery":100,"battery_low":false,"contact":true,"last_seen":"2023-01-12T08:11:25+00:00","linkquality":72,"tamper":false,"voltage":3000}'
Error 2023-01-12 08:11:31 Failed to configure 'contact_bathroom', attempt 2 (Error: Bind 0x00124b002512ec15/1 genPowerCfg from '0x00124b0024c04f51/1' failed (AREQ - ZDO - bindRsp after 10000ms) at Timeout._onTimeout (/app/node_modules/zigbee-herdsman/src/utils/waitress.ts:64:35) at listOnTimeout (node:internal/timers:559:17) at processTimers (node:internal/timers:502:7))
```
